### PR TITLE
ut improve coverage

### PR DESCRIPTION
**Pull request by Ywmet**
- Targeted PR against correct branch (see [CONTRIBUTING.md](https://github.com/okex/okexchain/blob/develop/CONTRIBUTING.md#pr-targeting))

- [ ] Linked to github-issue with discussion and accepted design OR link to spec that describes this work.
- [ ] Wrote tests
- [ ] Updated relevant documentation (`docs/`)
- [ ] Added a relevant changelog entry to the `Unreleased` section in `CHANGELOG.md`
- [ ] Reviewed `Files changed` in the github PR explorer

______

For Admin Use:
- Added appropriate labels to PR (ex. wip, ready-for-review, docs)
- Reviewers Assigned
- Squashed all commits, uses message "Merge PR #XYZ: [title]" ([coding standards](https://github.com/okex/coding/blob/master/README.md#merging-a-pr))

### CODE REVIEW

File: x/erc20/keeper/evm_log_handler_test.go [hunk 1]
```diff
@@ -88,6 +89,30 @@ func (suite *KeeperTestSuite) TestSendToIbcHandler() {
...
 		},
+		{
+			"denomination trace not found",
+			func() {
+				amount := sdk.NewInt(100)
+				suite.app.Erc20Keeper.SetContractForDenom(suite.ctx, validDenom, contract)
+				coin := sdk.NewCoin(validDenom, amount)
+				err := suite.MintCoins(sdk.AccAddress(contract.Bytes()), sdk.NewCoins(coin))
+				suite.Require().NoError(err)
+				suite.ctx.SetIsCheckTx(false)
```
Comment (xzav3r) [inline x/erc20/keeper/evm_log_handler_test.go]: `CheckTx` and `TraceTx` are both `false` by default，why should it be setted explicitly?

File: x/erc20/keeper/evm_log_handler_test.go [hunk 2]
```diff
@@ -88,6 +89,30 @@ func (suite *KeeperTestSuite) TestSendToIbcHandler() {
...
+				suite.ctx.SetIsTraceTx(false)
+
+				balance := suite.GetBalance(sdk.AccAddress(contract.Bytes()), validDenom)
+				suite.Require().Equal(coin, balance)
+
+				input, err := keeper.SendToIbcEvent.Inputs.Pack(
```
Comment (xzav3r) [inline x/erc20/keeper/evm_log_handler_test.go:147]: err should be checked

File: x/erc20/keeper/evm_log_handler_test.go [hunk 3]
```diff
@@ -162,6 +187,32 @@ func (suite *KeeperTestSuite) TestSendNative20ToIbcHandler() {
...
+				suite.app.Erc20Keeper.SetContractForDenom(suite.ctx, validDenom, contract)
+				coin := sdk.NewCoin(validDenom, amount)
+				err := suite.MintCoins(sdk.AccAddress(contract.Bytes()), sdk.NewCoins(coin))
+				suite.Require().NoError(err)
+				suite.ctx.SetIsCheckTx(false)
+				suite.ctx.SetIsTraceTx(false)
+				balance := suite.GetBalance(sdk.AccAddress(contract.Bytes()), validDenom)
+				suite.Require().Equal(coin, balance)
+				suite.app.TransferKeeper.SetParams(suite.ctx, types2.Params{true, true})
+				input, err := keeper.SendNative20ToIbcEvent.Inputs.Pack(
```
Comment (xzav3r) [inline x/erc20/keeper/evm_log_handler_test.go:248]: err should be checked

File: x/erc20/keeper/keeper_test.go [hunk 4]
```diff
@@ -146,6 +146,31 @@ func (suite *KeeperTestSuite) TestDenomContractMap() {
...
+				r := keeper.DeleteContractForDenom(suite.ctx, denom1)
+				suite.Require().Equal(r, false)
+				err := keeper.SetContractForDenom(suite.ctx, denom1, externalContract)
+				suite.Require().NoError(err)
+				r = keeper.DeleteContractForDenom(suite.ctx, denom1)
+				suite.Require().Equal(r, true)
+			},
+		},
+		{
+			"success, get contracts",
```
Comment (xzav3r) [inline x/erc20/keeper/keeper_test.go]: maybe it is better to rename it to `success, multiple denoms map to different contracts`

**Review by xzav3r — APPROVED** [review]

**Review by lcmmhcc — APPROVED** [review]

**Pull request merged**